Porting visualization and adding demo videos

suhaani-agarwal opened the pull request:
porting the following visualization from https://rcdata.nau.edu/genomic-ml/animint-gallery/ to https://animint.github.io/gallery/

 - 9, 18 (similar visualizations hence ported together) - [Changepoint detection ROC curve alignment problem](https://suhaani-agarwal.github.io/ROC-curve-alignment/) , old viz : [link](https://rcdata.nau.edu/genomic-ml/animint-gallery/2021-10-21-curveAlignment/index.html)

 - 22 - [Figure loss small data viz]( https://suhaani-agarwal.github.io/figure-loss-small-data/) old viz - [link](https://rcdata.nau.edu/genomic-ml/animint-gallery/2019-02-04-Full-path-of-changepoint-models-and-cost-functions/index.html)

I have added video for the following viz : 
[Montreal Bikes Jan 2016](https://suhaani-agarwal.github.io/montreal-bikes-Jan2016/)

here is the preview in my forked version of the gallery : https://suhaani-agarwal.github.io/gallery/

suhaani-agarwal committed:
update 2025-06-18 14:16:23.746985

suhaani-agarwal committed:
Merge remote-tracking branch 'upstream/gh-pages' into gh-pages

suhaani-agarwal committed:
update 2025-08-25 19:11:41.00804

tdhock commented:
the new data viz look good but there is a missing png image, see below.
<img width="1155" height="967" alt="image" src="https://github.com/user-attachments/assets/7d5519a9-aff5-4f52-a170-c1fcc5f9074a" />
can it be fixed by pulling master and updating? but it looks like you already tried that?

suhaani-agarwal committed:
update 2025-08-26 12:49:01.288519

suhaani-agarwal committed:
update 2025-08-26 12:53:23.300208

suhaani-agarwal committed:
update 2025-08-26 13:01:16.701822

suhaani-agarwal commented:
There were a few issues in the way the last two visualisations were ported. I’ve fixed the port for biplab-sutradhar/figure-candidates-interactive in my fork (https://suhaani-agarwal.github.io/gallery/).

It looks like meta.csv may have been edited manually earlier, which caused a mismatch between the actual source title ("Animint compiler with for loops") and the gallery title ("Data viz with 206 selectors"). Right now, the main repo shows "Data viz with 206 selectors," but after fixing the port by entering the repo in repos.txt, and removing the original entry from meta.csv , the gallery picked up "Animint compiler with for loops" (since that’s the title in the source code).

When I updated the gallery using `animint2::update_gallery()`, I also noticed inconsistencies between repo.txt (biplab-sutradhar/Data-viz-with-506-selectors) and the image file (biplab-sutradhar/Data-viz-with-206-selectors.png), which is why the image link broke. I’ve corrected this on my fork , but to fix this port completely , @biplab-sutradhar could you please update the source code so that the title is "Data viz with 206 selectors" and then re-run: animint2pages(viz, "<repo_name>")? and then rerun `animint2::update_gallery()` ?

tdhock commented:
thanks for the investigation.
Suhaani is right, you should edit the source/etc links in the R source code, not in the meta.csv file (which should be automatically created by update_gallery).

tdhock commented:
great thanks!
